docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+
+services:
+  server-node:
+    build: server/node/
+    environment:
+      - OPA_URL=http://opa:8181
+    ports:
+      - "4000:4000"
+    profiles:
+      - node
+
+  # TODO
+  server-java:
```

Comment:
I wonder if there is a convenient way with docker-compose to have several different possible server implementations that we swap between in a mutually exclusive way? I guess we could just comment and un-comment.

Reply:
That's what the --profile switch allows doing! We also need to set VARIANT.